src/main.rs

```diff
@@ -56,6 +56,10 @@ fn main() {
     simple_logger::init().unwrap();
 
     let args: Vec<String> = env::args().collect();
+    if args.len() != 2 {
+        println!("usage: run cargo <number-of-requests>");
+        std::process::exit(1);
```

Comment:
Add `use std::process` at top of file so you can change this to `process::exit(1);`